smack-core/src/main/java/org/jivesoftware/smack/sasl/gssApi/GssContext.java

```diff
@@ -0,0 +1,208 @@
+package org.jivesoftware.smack.sasl.gssApi;
+
+import org.ietf.jgss.*;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class GssContext implements GSSContext {
```

Comment:
Having classes with names that only differ in capitalization may cause issues on non-unix systems like windows.
While windows is not supported, I'd recommend using a different name for the interface anyway just to not complicate the situation further.

Reply:
I see that the GSSContext class is coming out of a library, so maybe this isn't an issue after all?